src/main/java/net/dv8tion/jda/api/entities/Invite.java

```diff
@@ -137,11 +137,19 @@
     /**
      * The type of this invite.
      *
-     * @return The invites's type
+     * @return The invite's type
      */
     @Nonnull
     Invite.InviteType getType();
 
+    /**
+     * The target type of this invite.
+     *
+     * @return The invite's target type
+     */
+    @Nonnull
+    Invite.TargetType getTargetType();
```

Comment:
What if there is no target?

Reply:
> What if there is no target?

If there is not target it will be TargetType.UNKNOWN

Reply:
That should be documented.

Reply:
Yeah added that in the last commit